examples/mapper_quickstart.ipynb

```diff
@@ -481,16 +481,27 @@
    "source": [
     "pipe = make_mapper_pipeline()\n",
     "\n",
-    "# Generate interactive plot\n",
-    "plot_interactive_mapper_graph(pipe, data, color_by_columns_dropdown=True)"
+    "# Generate interactive plots\n",
+    "fig = plot_interactive_mapper_graph(pipe, data, color_by_columns_dropdown=True)\n",
+    "display(fig)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**To see the interactive output, please download the notebook from [github](https://github.com/giotto-ai/giotto-tda/blob/master/examples/mapper_quickstart.ipynb) and execute it locally.**\n",
```

Comment:
This is a good effort but since the output does not show most of the key features of the interactive console, I'm not sure it is better than just saying it will only display locally.